action.yaml

```diff
@@ -24,6 +24,10 @@ inputs:
   tf_state_bucket:
     description: 'AWS S3 bucket to use for Terraform state. Defaults to `${org}-${repo}-{branch}-tf-state`'
     required: false
+  tf_state_bucket_destroy:
+    description: 'Force purge and deletion of S3 bucket defined. Any file contained there will be destroyed `stack_destroy` must also be `true`'
```

Comment:
the doc says: 
> `stack_destroy` must also be `true`

What if user set `tf_state_bucket_destroy=true` only without `stack_destroy`?
Will it provide any error message to the user or some hint, what will happen?

Reply:
nothing should happen because the destroy block is wrapped in:
```
if [[ $TERRAFORM_DESTROY = true ]] && [[ $TF_STATE_BUCKET_DESTROY = true ]]; then
```

a warn would be nice though like:
```
if [[ $TERRAFORM_DESTROY != true ]] && [[ $TF_STATE_BUCKET_DESTROY = true ]]; then
  echo "TF_STATE_BUCKET_DESTROY set to true, but TERRAFORM_DESTROY wasn't. Not destroying the state bucket ($TF_STATE_BUCKET)"
fi
```